coalib/coala_main.py

```diff
@@ -91,7 +94,9 @@ def run_coala(console_printer=None,
             if not section.is_enabled(targets):
                 continue
 
-            if force_show_patch:
+            if not autoapply:
+                section['default_actions'] = ''
+            elif force_show_patch:
                 section['default_actions'] = '*: ShowPatchAction'
```

Comment:
I think you need to make it clear in the doc string that `autoapply` and `force_show_patch` cannot be used together